src/containerManager.ts

```diff
@@ -32,39 +32,49 @@ export class ContainerManager {
   }
 
   manageContainer(): void {
+    if (this.containerStatus() == false){
+      const currentApp = getSelectedApp();
+      if (currentApp) {
+        this.statusBarManager.updateDevImageItem(DevImageStatus.stopped);
+        this.treeProvider.updateContainerLabel(DevImageStatus.stopped);
+        console.log(`Ledger: Container ${currentApp.containerName} not found, respawning it.`);
+        this.taskProvider.executeTaskByName("Update Container");
+      }
+    }
+  }
+
+  containerStatus(): boolean {
```

Comment:
Can't we get something a bit more explicit than a boolean as a return value here ?

Reply:
Done